components/bl-double-calendar-component/src/lib/date-fns/addDays.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
```

Comment:
Do we need all these functions in date-fns?

Reply:
Yes, these files are required in date-fns, but I'm not sure we need them for the component to work.